arm-insights/2016-03-01/swagger/insightsManagementClient_LogProfiles.json

```diff
@@ -64,21 +64,32 @@
             },
             "examples": {
               "application/json": {
-                "storageAccountId": "/subscriptions/6983c752-c9b8-48dd-b4d4-da739beb7e98/resourceGroups/Default-Storage-WestUS/providers/Microsoft.ClassicStorage/storageAccounts/armlogs1",
-                "serviceBusRuleId": "/subscriptions/6983c752-c9b8-48dd-b4d4-da739beb7e98/resourceGroups/Default-ServiceBus-WestUS/providers/Microsoft.ServiceBus/namespaces/myNamespace/RootManageSharedAccessKey",
-                "locations": [
-                  "eastus",
-                  "westus"
-                ],
-                "categories": [
-                  "Write",
-                  "Delete",
-                  "Action"
-                ],
-                "retentionPolicy": {
-                  "enabled": true,
-                  "days": 1
-                }
+                "value": [
+                  {
+                    "id":"/subscriptions/df602c9c-7aa0-407d-a6fb-eb20c8bd1192/providers/microsoft.insights/logprofiles/default",
+                    "type":null,
```

Comment:
I can remove the examples if they cause issues in the validation tools. Or I can put there valid values for the tools to succeed - these are examples after all and I know the calls work as they currently are - and file a task for Vinicius to fix the behavior or change the spec and the examples if needed.
NOTE: after this week, I will start handing off the specs to the people that own the services. They have more knowledge and reasons for why the services work as they do.